packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export type FeatureLinkListItemProps = LinkProps & {
+	label: ReactNode;
+	secondaryLabel: ReactNode;
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkListItem = ({
+	label,
+	secondaryLabel,
+	background = 'body',
+	...props
+}: FeatureLinkListItemProps) => {
+	const TextComponent = props.target == '_blank' ? TextLinkExternal : TextLink;
+
+	return (
+		<Box as="li" borderBottom paddingBottom={0.25}>
```

Comment:
You have an extra gap between items which becomes apparent on hover 
<img width="152" alt="image" src="https://github.com/user-attachments/assets/22590568-7915-4fc7-a0a6-cc30a6054234">